deadmattergame.ini

```diff
@@ -0,0 +1,7 @@
+[/Script/DeadMatter.DMGameSession]
+ServerName=AMP Powered Server
+MOTD=This server is powered by AMP
+PVP=true
+bIsHardcore=false
```

Comment:
Why is this included here, but not in the config manifest? How is this setting different to the `-hardcore` argument?

Reply:
Removed from Game.ini